.travis.yml

```diff
@@ -3,16 +3,11 @@
   sudo: required
   notifications:
     email: false
+  services:
+    - docker
   go:
     - 1.7
-  install: make deps
-  script: make .gitvalidation && make validate && make test && make test-skopeo
+  script: make test-pristine
```

Comment:
Do we really need to use an extra container for Travis? I’d expect Travis to give us a fresh VM every time.

(I am really mostly worried whether this would make the tests run significantly longer. I guess the only way to tell is to try in this PR.)